build.sbt

```diff
@@ -360,6 +360,7 @@ val sbtScriptedProjects = Seq[Project](
 lazy val root = (project in file("."))
   .settings(name := "lagom")
   .settings(runtimeLibCommon: _*)
+  .enablePlugins(JavaUnidocPlugin)
```

Comment:
https://github.com/sbt/sbt-unidoc#how-to-unify-both-scaladoc-and-javadoc seems to indicate that both `JavaUnidocPlugin` and `ScalaUnidocPlugin` are necessary. Not sure it's applicable to Lagom given all the custom settings in `project/Docs.scala` though.

Reply:
This is where things get tricky. My understanding of what `Doc.scala` is doing is basically forcing all docs to be _javadocs_; for this, I think unidoc (>=0.4) makes a distinction between enabling both `JavaUnidocPlugin` and `ScalaDocPlugin` (which I believe allows the command-issuer to publish both types of docs) and enabling `JavaUnidocPlugin` in subprojects and `GenJavadocPlugin` in the root, which forces all docs to be javadocs...

I might be wrong, but this is how I read the README.

Reply:
Associated docs [here](https://github.com/sbt/sbt-unidoc/blob/master/README.md#how-to-unify-javadoc)